### PR TITLE
`<Notificaiton/>`- Animator notification implementation

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Implementing `<Notification/>` using `Animator` library. This in order to resolve the animation issue we had. 

Resolves https://github.com/wix/wix-style-react/issues/4922

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5603/

File: src/Notification/Notification.js [hunk 1]
```diff
@@ -52,17 +47,34 @@ function mapChildren(children) {
   }, {});
 }
 
+const heightCalculation = e => {
+  const { height } = window.getComputedStyle(e.firstChild);
+  e.style.height = height;
+  return Number(height.replace(/px$/, ''));
+};
```
Comment (EgoziE) [inline src/Notification/Notification.js:54]:
```suggestion
const heightCalculation = element => {
  const height = element.firstChild.offsetHeight;
  element.style.height = `${height}px`;
  return height;
};
```

File: src/Notification/Notification.st.css [hunk 2]
```diff
@@ -4,27 +4,30 @@
...
 }
 
 .root {
   -st-states: theme(enum(standard, error, success,warning,premium )), type(enum(local, global, sticky));
   position: relative;
+  z-index: value(zIndex_NOTIFICATION);
+}
+
+.animatorContent > * {
+  position: relative;
```
Comment (EgoziE) [inline src/Notification/Notification.st.css:17]: Write a comment on ugly hack

File: src/Page/test/Page.e2eStory.js [hunk 3]
```diff
@@ -19,6 +13,15 @@ import { header, tail, fixedContent, content } from '../docs/PageChildren';
...
 
+import {
+  Page,
+  Card,
+  Box,
+  Notification,
+  PopoverMenu,
+  IconButton,
+} from 'wix-style-react';
+
```
Comment (EgoziE) [inline src/Page/test/Page.e2eStory.js:24]: can be removed

File: src/Notification/Notification.js (hunk 1 above)
Comment (EgoziE) [reply src/Notification/Notification.js:54]: I think this requires a comment

**Comment by TalZa1810** [comment]
failed on visual which is not related.

**Pull request merged**